### ISSUE TITLE
SNES Mayflash Adapter Issue

### ISSUE THREAD

**Issue by Jeebs23** [MiSTer-devel/Main_MiSTer#187]
I posted this in the Atari Forum but I figured I'd add it here as an official issue.

The issue looks to be with merging P1 and P2 together with this particular device. 

paulbnl mentioned that the VID of this adapter needs to be added to disable merging for Mayflash devices.

I'm not sure which of these three formats is needed but here is what I found for this device.

```
input_0e8f_3013_v3.map
HID\VID_0E8F&PID_3013&REV_0111&Col02
0e8f:3013
```
Is this something that can be done when possible please? If you need anything else from me please let me know.

**Comment by mlc4050** [comment]
Line 2733 of input.cpp can be changed to the following to support the Mayflash adapter:	
	if (input[i].vid == 0x289B || input[i].vid == 0x0E8F) continue;

**Comment by sorgelig** [comment]
i will add it later.

**Comment by mlc4050** [comment]
This was resolved in release 20200419. This issue can be marked as closed.
